onnxmltools/convert/common/_topology.py

```diff
@@ -683,7 +683,8 @@ def convert_topology(topology, model_name, doc_string, target_opset, targeted_on
             invalid_name.append(name)
         if name in tensor_inputs:
             onnx_input = tensor_inputs[name]  # type: Variable
-            if name in channel_first_inputs:
+            if name in channel_first_inputs or \
```

Comment:
Hummm, it looks not good. Why do you need extra conditions?

Reply:
to support the keras input name like 'input_1', instead of 'input_1:0'.